Arm/Cosim.lean

```diff
@@ -109,7 +109,8 @@ def arm_cosim_test (input : regState) : IO String := do
   let gprs'   := bitvec_to_hex_list input.gpr
   let flags'  := bitvec_to_hex input.nzcv
   let sfps'   := bitvec_to_hex_list input.sfp
-  let args    := ([inst'] ++ gprs' ++ [flags'] ++ sfps').toArray
+  let instructionStr := ([inst'] ++ gprs' ++ [flags'] ++ sfps')
+  let args    := [toString guid] ++ instructionStr |>.toArray
```

Comment:
```suggestion
  let args    := toString guid :: instructionStr |>.toArray
```
Seems more readable to me, but tastes differ

Also, what is instructionStr exactly?

Reply:
I elaborated on this in the new commit.